### PR TITLE
Add HiKey build instructions to README.md

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
     4. [QEMU](#44-qemu)
     4. [STMicroelectronics boards](#45-stmicroelectronics-boards)
     4. [Allwinner A80](#46-allwinner-a80)
+    4. [HiKey Board](#48-hikey-board)
```
Comment (vchong) [inline README.md]: Nitpick: 47 instead of 48

File: README.md [hunk 2]
```diff
@@ -565,6 +566,30 @@ $ /system/bin/tee-helloworld
 ```
 Enjoying OP-TEE on A80 board.
 
+---
+### 4.8 HiKey board
```
Comment (vchong) [inline README.md]: Nitpick: 4.7 instead 4.8

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: README.md (hunk 2 above)
Comment (jbech-linaro) [inline README.md]: Either we keep 4.8 here since MediaTek PR are using 4.7 or we have a race between commits.

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: README.md (hunk 2 above)
Comment (vchong) [inline README.md]: Oops, sorry. My bad. Didn't realize MTK is 4.7. Thanks for pointing it out.

**Comment by pascal-brand-st-dev** [comment]
#305 has just been merged, 4.7 being MTK.

**Comment by jforissier** [comment]
Rebased and review tags applied.

**Pull request merged**